# the id of tag detected
int32 id

# the pose of tag with respect to camera
geometry_msgs/Pose pose

# the coefficent of meters per pixel
float64 PixelCoef

# the detected tag center
geometry_msgs/Point center

# the coordinate of tag cent measured by depth camera
geometry_msgs/Point Center3d
